### ISSUE TITLE
[meetup] Backend doesn't fetch existing events for some groups

### ISSUE THREAD

**Issue by albertinisg** [grimoirelab/perceval#261]
Lately I had the feeling that we were not retrieving all the events, and I thought it was due to the rate limit. Today I did checked some groups and found that, even if the group exists, is reachable and contains events, perceval does not get any. The following group https://www.meetup.com/Blockchain-Meetup-Freiburg for example, contains some, but running with perceval:
```bash
$ perceval -g meetup "Blockchain-Meetup-Freiburg" -t XXXXXXXXXXXXXXXXXXXXXXXX
[2017-12-22 16:22:44,973 - root - INFO] - Sir Perceval is on his quest.
[2017-12-22 16:22:44,976 - perceval.backends.core.meetup - INFO] - Fetching events of 'Blockchain-Meetup-Freiburg' group from 1970-01-01 00:00:00+00:00 to --
[2017-12-22 16:22:44,977 - perceval.backends.core.meetup - DEBUG] - Meetup client calls resource: Blockchain-Meetup-Freiburg/events?fields=event_hosts,featured,group_topics,plain_text_description,rsvpable,series&status=cancelled,upcoming,past,proposed,suggested,draft params: {'order': 'updated', 'scroll': 'since:1970-01-01T00:00:00.000Z', 'page': 200, 'key': 'XXXXXXXXXXXXXXXXXXXXXXXX', 'sign': ('true',)}
[2017-12-22 16:22:44,979 - urllib3.connectionpool - DEBUG] - Starting new HTTPS connection (1): api.meetup.com
[2017-12-22 16:22:45,208 - urllib3.connectionpool - DEBUG] - https://api.meetup.com:443 "GET /Blockchain-Meetup-Freiburg/events?fields=event_hosts,featured,group_topics,plain_text_description,rsvpable,series&status=cancelled,upcoming,past,proposed,suggested,draft&order=updated&scroll=since%3A1970-01-01T00%3A00%3A00.000Z&page=200&key=XXXXXXXXXXXXXXXXXXXXXXXX&sign=true HTTP/1.1" 200 2
[2017-12-22 16:22:45,209 - perceval.client - DEBUG] - Rate limit: 29
[2017-12-22 16:22:45,210 - perceval.client - DEBUG] - Rate limit reset: 10
[2017-12-22 16:22:45,218 - perceval.backends.core.meetup - INFO] - Fetch process completed: 0 events fetched
[2017-12-22 16:22:45,218 - root - INFO] - Sir Perceval completed his quest.
```

**Comment by sduenas** [comment]
Looks like a problem of their API. When `draft` status is using as filter it causes to get an empty list of events.

I filed a bug report on their tracker. More info here: meetup/api#260

**Comment by sduenas** [comment]
This was partially fixed in #267. It removes the draft status but the bug is still open in Meetup. We will reopen this as soon as it gets fixed in Meetup API.
